pkg/clients/aws/route53.go

```diff
@@ -293,6 +298,149 @@ func NewClient(kubeClient client.Client, secretName, namespace, region string) (
+		cm := &corev1.ConfigMap{}
+		err := kubeClient.Get(context.TODO(), types.NamespacedName{
+			Name:      aaov1alpha1.DefaultConfigMap,
+			Namespace: aaov1alpha1.AccountCrNamespace,
+		}, cm)
+
+		if err != nil {
+			return nil, fmt.Errorf("error getting aws-account-operator configmap to get the sts jump role: %v", err)
+
+		}
+
+		stsAccessARN := cm.Data["sts-jump-role"]
+		if stsAccessARN == "" {
+			return nil, fmt.Errorf("aws-account-operator configmap missing sts-jump-role: %v", aaov1alpha1.ErrInvalidConfigMap)
+		}
+
+		// Get STS Creds
+		secret := &corev1.Secret{}
+		err = kubeClient.Get(context.TODO(),
+			types.NamespacedName{
+				Name:      awsCredsSecretName,
+				Namespace: namespace,
+			},
+			secret)
+
+		if err != nil {
+			return nil, err
+		}
+
+		accessKeyID, ok := secret.Data[awsCredsSecretIDKey]
+		if !ok {
+			return nil, fmt.Errorf("AWS credentials secret %v did not contain key %v",
+				secretName, awsCredsSecretIDKey)
+		}
+
+		secretAccessKey, ok := secret.Data[awsCredsSecretAccessKey]
+		if !ok {
+			return nil, fmt.Errorf("AWS credentials secret %v did not contain key %v",
+				secretName, awsCredsSecretAccessKey)
+		}
+
+		awsConfig.Credentials = credentials.NewStaticCredentials(
+			strings.Trim(string(accessKeyID), "\n"),
+			strings.Trim(string(secretAccessKey), "\n"),
+			"",
+		)
+
+		s, err := session.NewSession(awsConfig)
+		if err != nil {
+			return nil, err
+		}
+
+		hiveAwsClient := sts.New(s)
+
+		jumpRoleCreds, err := getSTSCredentials(hiveAwsClient, stsAccessARN, "", "certmanOperator")
+		if err != nil {
+			return nil, err
+		}
+
+		jumpConfig := &aws.Config{
+			Region:     aws.String(region),
+			MaxRetries: aws.Int(clientMaxRetries),
+			Retryer: awsclient.DefaultRetryer{
+				NumMaxRetries:    retryerMaxRetries,
+				MinThrottleDelay: retryerMinThrottleDelaySec * time.Second,
+			},
+			Credentials: credentials.NewStaticCredentials(
+				*jumpRoleCreds.Credentials.AccessKeyId,
+				*jumpRoleCreds.Credentials.SecretAccessKey,
+				*jumpRoleCreds.Credentials.SessionToken,
+			),
+		}
+
+		js, err := session.NewSession(jumpConfig)
+		if err != nil {
+			return nil, err
+		}
+
+		jumpRoleClient := sts.New(js)
+
+		// Get Account's STS role from AccountClaim
+		accountClaim := &aaov1alpha1.AccountClaim{}
+		err = kubeClient.Get(context.TODO(), types.NamespacedName{
+			Name:      clusterDeploymentName,
+			Namespace: namespace,
+		}, accountClaim)
+		if err != nil {
+			return nil, err
+		}
+
+		if accountClaim.Spec.ManualSTSMode {
+			if accountClaim.Spec.STSRoleARN == "" {
+				return nil, fmt.Errorf("STSRoleARN missing from AccountClaim %v", accountClaim.Name)
+			}
+
+		}
+
+		// TODO: use the accountClaim.Spec.STSExternalID here once I move to the new aws-account-operator api
+		customerAccountCreds, err := getSTSCredentials(jumpRoleClient, accountClaim.Spec.STSRoleARN, "", "RH-Account-Initilization")
+
+		if err != nil {
+			return nil, err
```

Comment:
Similar message here `fmt.Errorf("unable to assume customer role %s: %v", accountClaim.Spec.STSRoleARN, err)`